addons/web/static/src/js/views/graph/graph_view.js

```diff
@@ -138,7 +138,6 @@ var GraphView = AbstractView.extend({
         this.loadParams.measure = measure || '__count__';
         this.loadParams.groupBys = groupBys;
         this.loadParams.fields = this.fields;
-        this.loadParams.comparisonDomain = params.comparisonDomain;
```

Comment:
what about this commit? It this just a bit of cleaning?

Reply:
yes

Reply:
could be done in owl graph renderer